packages/oauth-twitter/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import * as oauth from 'oauth';
+
+export interface AccountsOauthTwitterOptions {
+  key: string;
+  secret: string;
+}
+
+class AccountsOauthTwitter {
+  private options: AccountsOauthTwitterOptions;
+
+  constructor(options: AccountsOauthTwitterOptions) {
+    this.options = options;
+  }
+
+  public authenticate(params) {
+    const oa = new oauth.OAuth(
```

Comment:
Should maybe create the oauth object on the constructor. Also maybe use [purest?](https://github.com/simov/purest)